pkg/jx/cmd/import.go

```diff
@@ -208,6 +211,41 @@ func (options *ImportOptions) Run() error {
 		return nil
 	}
 
+	if options.DetectLangOnly {
```

Comment:
I think this should really be extracted into a dedicated method. Also there seems to be a fair bit of duplicated logic in this method. I am wondering whether this somehow could be unified?